src/components/query-controls.js

```diff
@@ -255,11 +257,33 @@ class QueryControls extends Component {
 		return (
 			<>
 				{ enableSpecific && (
-					<ToggleControl
-						checked={ specificMode }
-						onChange={ onSpecificModeChange }
-						label={ __( 'Choose Specific Posts', 'newspack-blocks' ) }
-					/>
+					<BaseControl
+						label={ __( 'Type', 'newspack-blocks' ) }
```

Comment:
Non-blocking suggestion:

```suggestion
						label={ __( 'Mode', 'newspack-blocks' ) }
```